scripts/project.js

```diff
@@ -0,0 +1,50 @@
+// Constructor function for portfolio item
+(function(module){
+  function PortfolioItem (opts) {
+    for (key in opts) {
+      this[key] = opts[key];
+    }
+  }
+
+  PortfolioItem.prototype.buildThumbnails = function() {
```

Comment:
refactor buildThumbnails and updateDetailModal into a single function. You can pass the source parameter as an argument.